conf/conf.py

```diff
@@ -18,6 +18,8 @@
 PATCH_SYSTEMS_FILE = "/tmp/patch_systems.csv"
 VULNERABILITY_SYSTEMS_FILE = "/tmp/vulnerability_systems.csv"
 ADVISOR_FILE = "/tmp/advisor_systems.csv"
+STALE_HOSTS_LIST = "/tmp/stale_hosts.csv"
```

Comment:
@rafaelguerra01 there is a recent change on this file. I believe a git rebase will be good. We can check this out together.